packages/client/src/components/AppHeader/AppHeader.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import gql from 'graphql-tag';
+import { useQuery } from '@apollo/react-hooks';
+import { Loader, Icon } from '@code-mentoring/ui';
+import { User } from '@code-mentoring/api';
+import { ErrorMessage } from '@code-mentoring/ui/components/ErrorMessage/ErrorMessage';
+import LogoMark from '../../images/logo-mark.svg';
+import { DropdownMenu } from '../DropdownMenu/DropdownMenu';
+
+// TODO: Replace user profile picture when #27 is completed.
+// TODO: Replace the class icon with the actual class the student is
+// TODO: Add conditionals for things that are supposed to appear depending on the page (ie. points, course)
+
+
+const meQuery = gql`
+  query {
+    me {
+      firstName
+      lastName
+      email
+      id
+    }
+  }
+`;
+
+export const AppHeader: React.FC = () => {
+  const { data, loading, error } = useQuery<{ me: User }>(meQuery);
```

Comment:
You can user the ME container here that already does this work :) https://github.com/code-mentoring/learn/blob/develop/packages/client/src/containers/Me.container.ts